packages/client/components/PokerDimensionValueControl.tsx

```diff
@@ -201,6 +247,15 @@ const PokerDimensionValueControl = (props: Props) => {
           </>
         )}
       </Control>
+      {addMissingJiraFieldModalPortal(
+        <AddMissingJiraFieldModal
+          finalScore={finalScore}
```

Comment:
Actually, this was a bad choice as the majority of this data can be managed by Relay. I've changed it to look this way
```
        <AddMissingJiraFieldModal
          pendingScore={pendingScore}
          stage={stage}
          closePortal={closeAddMissingJiraFieldModal}
        />
```
where `stage` is just a Relay fragment
```
  stage: graphql`
    fragment AddMissingJiraFieldModal_stage on EstimateStage {
      id
      meetingId
      teamId
    }
  `
```

Regarding object vs primitives, there's no easy answer here. I personally think it's better to pass objects when the number of props starts growing (and group the props if possible) just to make the code more readable. When passing objects which will be used as hooks dependencies it's critical to remember to _not_ recreate them on every render because it might cause infinite loops. It's not the case when using Relay 🤗